osidb/migrations/0112_flaw_local_updated_dt_data.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from django.conf import settings
+from django.db import migrations
+from django.db.models import Max
+
+from osidb.core import set_user_acls
+
+
+logger = logging.getLogger(__name__)
+
+
+BATCH_SIZE = 1000
+
+
+def forwards_func(apps, schema_editor):
```

Comment:
Have you had a chance to test this migration against a db dump of a non trivial size ? I would not want the first test to be data migration to prod ;)

Reply:
I tested it against a db that is 1/4th the size and it takes ~1s per 1000 flaws, based off of production data, but as noted in the TODO in the PR description I might improve it further for faster deployment (but even in its current state it should not be too slow, definitely under the 30m deadline)